docs/introduction/requirements.md

```diff
@@ -0,0 +1,46 @@
+# Requirements
+
+## Quantitative values
+
+### Real time tracking of resources
+
+*present*
+
+It involves `vf:currentQuantity` as well as *available quantity*. They might differ for example when ongoing *use* event(s) involves certain quantity of resource. *Available quantity* would equal `vf:currentQuantity` minus all the quantities currently involved in *use* event(s).
```

Comment:
Also possibly accept-improve processes, transportation processes... maybe others?

Reply:
https://valueflo.ws/introduction/flows.html#actions
* *load* and *unload* already decrement and increment they will get reflected on `currentQuantity`
* *accept* and *improve* at this moment don't increment or increment which to stay honest seems inconsistent to me

All four above differ from *use* in respect that acting as pair one can decrement and other one can increment the resource. Since we have only one flow with *use* it can't both increment and decrement the resource.

Reply:
I see what you are saying with load/unload.

accept / improve were created for moving the same resource through one or more stages.  Originally they were for translating, which involved initial translation, proofreading, editing, etc. of a document - I forget the exact stages and sequence.  In any case, it is the same resource in and out, which requires an additional piece of identification to not create cycles - we used process type.  The other use case we had was Sensorica QT (quality testing) of a part that had been produced.